src/core_plugins/kibana/public/context/api/utils/__tests__/fields.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
```

Comment:
Would you mind switching from Chai to expect.js? The vast majority of our tests are written with expect.js. It looks like Chai was pulled in with Timelion which was originally an external plugin. We have an open issue about [moving to Chai](https://github.com/elastic/kibana/issues/5894) but until then it would be better to stick to expect.js.